### PR TITLE
Added swagger to the project (see backend/README.md)

### PR DESCRIPTION
**Pull request by tjanowsk**
Used swagger on the server.

Server's attack sharply rose.
Server is confused.

### CODE REVIEW

**Comment by floyd924** [comment]
Would it be worth adding the required annotation to a, existing hello world endpoint so as to show the next developer how to annotate future endpoints?

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ build
 .project
 .settings
 *.iml
+.vscode
```
Comment (AdrianBorg) [inline .gitignore:9]: You're right here in that we probably don't want to add he .vscode dir into out repo. Maybe i should just document what should be included in the readme for PR #43, and also add the .vscode to the gitignore there.

Comment (tjanowsk) [reply .gitignore:9]: I added it before looking at your PR. Since it contains only 1 json file I'm not bothered by it.

Comment (AdrianBorg) [reply .gitignore:9]: Do we want to remove it from the gitignore, or the repo then?

**Pull request merged**